src/blockdev.rs

```diff
@@ -191,6 +191,14 @@ pub fn reread_partition_table(file: &mut File) -> Result<()> {
             Ok(_) => break,
             Err(err) => {
                 if retries == 0 {
+                    let status = std::process::Command::new("kpartx")
+                        .arg("-u")
```

Comment:
Do we also need to pass `-s`?

Reply:
i'm not sure. my expectation is that after successful copying of metal-image to disk we don't have to wait until partitions are created,because by that time disk should be ready, and we only have to re-read partition table.

Reply:
Sure, but we need to wait for the device nodes to appear before using them.  `udev_settle()` tries to do that, but there's currently still a possible race condition.  It should be possible to address that generically with #155 or something like it, so I'm okay omitting `-s` if we want to exercise `udev_settle()` uniformly in all paths.

Reply:
during testing i had extra calls to `udevadm trigger -c add` for `subsystems and devices` within `udev_settle()`, but it looked like an overhead, so it was removed.
What do you think about using it?

Reply:
As I understand it, `udevadm trigger` shouldn't generally be used in production.  udev events will already be generated by the new device nodes; we just need to make sure we're waiting for them appropriately.

Reply:
Agree. Is there smth else i have to modify/improve ?

Reply:
I think we're okay on the udev part.  Items 1-4 of https://github.com/coreos/coreos-installer/pull/211#issuecomment-610705093 should still be addressed.